scripts/Jenkinsfile

```diff
@@ -0,0 +1,172 @@
+				git_remote_add("hcc_fork/clang-tools-extra","clang-tools-extra_fork", "ssh://git@github.com/${ userID}/clang-tools-extra.git")
+				git_remote_add("hcc_fork/clang-tools-extra","clang-tools-extra", "ssh://git@github.com/llvm-mirror/clang-tools-extra.git")
+				git_remote_add("hcc_fork/clang","clang_fork", "ssh://git@github.com/${ userID}/hcc-clang-upgrade.git")
+				git_remote_add("hcc_fork/clang","clang", "ssh://git@github.com/RadeonOpenCompute/clang")
+			}
+		}
+		
+		if (startStage <= 4) { 
+			stage ('Stage 4:Checkout latest master ROCM-Device-Libs') {
+				sh (script: "cd hcc_fork/rocdl; git fetch --all; git checkout origin/master")
+			}
+		}
+		
+		if (startStage <= 5) { 		
+			stage ('Stage 5: Merge amd-common  commits') {
+			    
+			    if (!fileExists("merge-branch-name") ) { 
+			    	writeFile file: "merge-branch-name", text: "${ merge_name}"
+				} 
+				merge_name = readFile "merge-branch-name"
+				
+				dir("hcc_fork") {
+					git_config()
+					git_merge("lld", merge_name, "origin/amd-hcc", "origin/amd-common")
+					git_merge("compiler",merge_name,"origin/amd-hcc", "origin/amd-common")
+					git_merge("compiler-rt", merge_name, "origin/amd-hcc", "--no-ff compiler-rt/master")
+					git_merge("clang-tools-extra", merge_name, "origin/amd-hcc", "clang-tools-extra/master")
+					git_merge("clang", merge_name, "origin/clang_tot_upgrade", "--no-ff clang/amd-common")
+				}
+			}		
+		}
+		
+		if (startStage <= 6) { 
+			stage ('Stage 6: Build merged HCC') {
+				sh (script: "mkdir -p build; cd build; cmake -DCMAKE_BUILD_TYPE=Release ../hcc; make -j10")
+			}
+		}
+		
+		if (startStage <= 7) { 
+			stage ('Stage 7: Full sanity tests on merged HCC') {
+				sh (script: "cd build; make test")
+			}
+		}
+		
+		if (startStage <= 8) { 		
+			stage ('Stage 8: Quick sanity tests on merged HCC') {
+				sh (script: "/opt/rocm/bin/hcc `/opt/rocm/bin/clamp-config --build --cxxflags --ldflags` -lm hcc/tests/Unit/AmpMath/amp_math_cos.cpp; ./a.out ")
+				sh (script: "/opt/rocm/bin/hcc `/opt/rocm/bin/hcc-config --build --cxxflags --ldflags` -lhc_am hcc/tests/Unit/GridLaunch/glp_const.cpp; ./a.out ")
+				sh (script: "/opt/rocm/bin/hcc `/opt/rocm/bin/hcc-config --build --cxxflags --ldflags` hcc/tests/Unit/HC/hc_atomic_add_global.cpp ; ./a.out ")
+			}
+		}
+		
+		if (startStage <= 9) { 
+			stage('Stage 9: Push and Create Pull Requests') {
+			
+				if (!fileExists("merge-branch-name") ) { 
+			    	writeFile file: "merge-branch-name", text: "${ merge_name}"
+				} 
+				merge_name = readFile "merge-branch-name"
+				
+				dir ("hcc_fork/clang") {
+					git_config()
+					sh (script: "git push clang_fork ${merge_name}:${merge_name}")
+				}
+				dir ("hcc/clang") {
+					git_config() 
+					sh (script: "git request-pull ssh://git@github.com/${ userID}/hcc-clang-upgrade clang_tot_upgrade")
+				}
+				
+			//	sh (script: "cd hcc_fork/lld; git push lld_fork ${merge_name}:${merge_name}")
```

Comment:
@david-salinas could you elaborate why these forks aren't pushed and have PRs filed?

Reply:
So, I haven't currently had success in getting the "git request-pull" commands to work.  I'm thinking we may just have people manually create them in GitHub through the web UI - unless someone already knows the correct syntax for these commands.  I just have them commented out, for now, until we resolve this.